client/styles/components/_nav.scss

```diff
@@ -19,6 +19,7 @@
   display: flex;
   flex-direction: row;
   justify-content: flex-end;
+  min-width: 10%;
```

Comment:
I don't think this line would come into play it all.  Correct me if I'm wrong.  I think you should delete this.